source/install/self-managed-changelog.md

```diff
@@ -6,10 +6,96 @@ See the [changelog in progress](https://bit.ly/2nK3cVf) for the upcoming release
 
 Latest Mattermost Releases:
 
+- [Release v7.11 - Feature Release](#release-v7-11-feature-release)
 - [Release v7.10 - Feature Release](#release-v7-10-feature-release)
 - [Release v7.9 - Feature Release](#release-v7-9-feature-release)
 - [Release v7.8 - Extended Support Release](#release-v7-8-extended-support-release)
-- [Release v7.1 - Extended Support Release](#release-v7-1-extended-support-release)
+
+## Release v7.11 - [Feature Release](https://docs.mattermost.com/upgrade/release-definitions.html#feature-release)
+
+**Release day: May 16, 2023**
+
+### Important Upgrade Notes
+
+ - In v7.11, the following repositories are merged into one: ``mattermost-server``, ``mattermost-webapp``, ``focalboard`` and ``mattermost-plugin-playbooks``. Developers should read the updated [Developer Guide](https://developers.mattermost.com/contribute/developer-setup/) for details. **Playbooks and Boards are now core parts of the product and cannot be disabled**.
+ - Fixed an issue caused by a migration in the previous release. Query takes around 11ms on a PostgreSQL 14 DB t3.medium RDS instance. Locks on the preferences table will only be acquired if there are rows to delete, but the time taken is negligible.
+
+**IMPORTANT:** If you upgrade from a release earlier than v7.10, please read the other [Important Upgrade Notes](https://docs.mattermost.com/upgrade/important-upgrade-notes.html).
+
+### Improvements
+
+#### User Interface (UI)
+ - Enabled Work Templates in the Onboarding checklist and the `+` Menu.
+ - Added a **Mattermost Marketplace** option to the bottom of the apps bar. The option is visible when the Marketplace is enabled, and the user has ``SYSCONSOLE_WRITE_PLUGINS`` permissions.
+ - Added an **Add channels** button to the bottom of the left-hand sidebar to make the action more obvious for users who want to create or join channels.
+ - Removed the Webapp Build Hash from **Main Menu > About Mattermost** since it is now identical to Server Build Hash.
+ - Replaced the ``compass-components`` icon component with ``compass-icons``.
+ - Added “hours ahead” timezone details to the user profile popover.
+
+#### Administration
+ - Self-hosted admins can now define a separate shipping address.
+ - Added updates to the trial request forms to allow for a more tailored trial experience.
+ - Added a form page to capture company name data, and added the invite members section to the Preparing Workspace screen for self-hosted installations.
+ - Adds the ability to expand seats in-product for self-hosted servers.
+ - Added a new section in the **System Console** for products.
```

Comment:
We should probably explain this a bit better because at first I though products was like SKUs but I believe this is referring to Boards and Playbooks?